check_perccli

```diff
@@ -134,11 +135,15 @@ def main():
             log.debug('C%s VD%s: %s (%s)' % (c, v, vd.get('TYPE'), vd_state))
             msg.append('VD%s: %s%s' % (v, vd_state, ' (Inconsistent)' if vd_consist == 'No' else ''))
 
-        battery_state = controller.get('Response Data').get('BBU_Info')[0].get('State')
-        if battery_state != 'Optimal':
-            exit_code = 2
-        log.debug('C%s BBU (%s)' % (c, battery_state))
-        msg.append('BBU: %s' % battery_state)
+        bbu = controller.get('Response Data').get('BBU_Info')
+
+        # Some RAID cards like H330 do not have a battery so make this optional
+        if (bbu != None):
+          battery_state = bbu[0].get('State')
+          if battery_state != 'Optimal':
```

Comment:
I highly recommend linting the code before submitting PR:
```
$ pycodestyle --max-line-length=120 check_perccli
check_perccli:88:121: E501 line too long (154 > 120 characters)
check_perccli:89:1: E101 indentation contains mixed spaces and tabs
check_perccli:89:1: W191 indentation contains tabs
check_perccli:89:121: E501 line too long (129 > 120 characters)
check_perccli:91:1: E101 indentation contains mixed spaces and tabs
check_perccli:141:17: E711 comparison to None should be 'if cond is not None:'
check_perccli:142:11: E111 indentation is not a multiple of four
check_perccli:143:11: E111 indentation is not a multiple of four
check_perccli:144:15: E111 indentation is not a multiple of four
check_perccli:145:11: E111 indentation is not a multiple of four
check_perccli:146:11: E111 indentation is not a multiple of four
```

Reply:
Alright, thanks.